.eslintignore

```diff
@@ -0,0 +1,6 @@
+/dist
+/.nuxt
+/.output
+/public/assets/fonts
+/functions
+/translations/
```

Comment:
it might be missing a `.yarn` here too?

Reply:
I guess here it's fine, we don't want to lint `.yarn` folder :D